### PR TITLE
Added initial set of integration tests for event GET API endpoints

### PR DESCRIPTION
**Pull request by kevhuang**

### CODE REVIEW

**Comment by dmsakamoto** [comment]
:+1: Thanks for deleting local thirdPartyServices stuff and doing the cleanup

File: server/test/events.spec.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  });
+
+  it('Should return the correct event on GET ' + basePath + '/:eventId for a valid eventId', function(done) {
+    request(app)
+      .get(basePath + '/' + eventId)
+      .set('Accept', 'application/json')
+      .expect('Content-Type', /json/)
+      .expect(200)
+      .expect(function(res) {
+        if (res.body.id !== eventId)
```
Comment (ekong2) [inline server/test/events.spec.js:67]: also check the res.body.name and location? makes it a bit more rigorous. up to you

**Comment by ekong2** [comment]
rigorous or not, still good though

File: server/test/events.spec.js (hunk 1 above)
Comment (kevhuang) [inline server/test/events.spec.js:67]: Thanks, I'll add that later

**Pull request merged**